lib/escape.go

```diff
@@ -0,0 +1,32 @@
+package lib
+
+import "strings"
+
+// MarkdownEscape wraps some text `t` in triple backticks (escaping any backtick
+// inside the message), limiting the length of the message to `n` runes (inside
+// the single preformatted block). The text is trimmed before escaping.
+// Backticks are escaped and thus count as two runes for the purpose of the
+// truncation.
```

Comment:
Could you add an example to this comment?
```
e.g. x -> y
```

Reply:
I went ahead and added an example test with a small handful of test cases.